code/modules/research/nodes/illegal.dm

```diff
@@ -46,8 +46,8 @@
 	unlocks_designs = list(/datum/design/research/item/chameleon_kit)
 
 /datum/technology/freedom_implant
-	name = "Glass Case- 'Freedom'"
```

Comment:
How this is related to shades change?

Reply:
Adds possibility to make some illegal shades through R&D? Lol.

Reply:
`/datum/technology/freedom_implant`

Reply:
Okay.

Reply:
Changed.